posts/packit-in-2024/index.md

```diff
@@ -0,0 +1,58 @@
+
+The second half of the year was spent finishing all the initiatives we’d worked on for some time. The long-awaited feature was not to require a git upstream for Fedora release automation. This wasn’t so simple because of the old decision to build Packit on the concept of related git repositories – one representing upstream and one downstream. (And maybe one for the source-git, but that’s a [story](https://packit.dev/source-git/status) for another time.) Luckily, all the hard work was done and many packages can now use Packit.
+
+Another feature and another change of Packit’s internal plumbings is the way to [avoid having divergent branching](https://packit.dev/posts/non-git-upstreams-support) when releasing to multiple Fedora (or EPEL) branches. Thanks to this, the commits between branches can now be shared and the history can be linear.
+
+This is still not all. Another huge feature spreading across multiple packages is [the support for side-tags](https://packit.dev/docs/fedora-releases-guide/releasing-multiple-packages). It’s a vast improvement and enabler for various package groups that need to control many packages together. And it’s finally there!
+
+And this is still not all for this year. We love collaborating on features and one such collaboration was/is with [Siteshwar Vashisht](https://fosstodon.org/@siteshwar) on integrating Packit with [OpenScanHub](https://openscanhub.dev/) so Packit users can run static analysis without much effort. Later during the year, we managed to properly report differential scans (so you can check only new findings.) If you are interested in this (or would like to know why one needs to care), look at Situ’s presentation on [DevConf.CZ](https://www.youtube.com/watch?v=rcuIvAttWgY&pp=ygUdZGV2Y29uZiBzaXRlc2h3YXIgb3BlbnNjYW5odWI%3D) or [Cauldron](https://www.youtube.com/watch?v=lp7UECNg7qY) conference. You can also visit Brussels next year and [František](https://mastodonczech.cz/@lachmanfrantisek) and [Siteshwar Vashisht](https://fosstodon.org/@siteshwar) should present the integration on the [CentOS Connect](https://connect.centos.org/).
+
+After all these big achievements and manysmall and big improvements, the [time has come to release Packit v1.0.0](https://packit.dev/posts/packit_1_0_0_action_required). Everything is now prepared, and we are giving people time to update their configuration files. (Since we love automation and pull request-based workflows, the affected users should have received a pull request with the changes required.)
```

Comment:
```suggestion
After all these big achievements and many small and big improvements, the [time has come to release Packit v1.0.0](https://packit.dev/posts/packit_1_0_0_action_required). Everything is now prepared, and we are giving people time to update their configuration files. (Since we love automation and pull request-based workflows, the affected users should have received a pull request with the changes required.)
```